frontend/src/Student/components/Portfolios.js

```diff
@@ -0,0 +1,53 @@
+import PropTypes from 'prop-types'
+import React, { Component } from 'react'
+import styled from 'styled-components'
+
+import PortfolioCard from '../components/PortfolioCard'
+import Loading from '../../shared/components/Loading'
+
+const NoShowsContainer = styled.div`
+  font-size: large;
+  left: 50%;
+  position: fixed;
+  top: 50%;
+  transform: translate(-50%, -50%);
+`
+class Portfolios extends Component {
+  renderShows = shows => {
+    if (shows.length === 0) {
+      return (
+        <NoShowsContainer>
+          The portfolio period is not currently open. Check back soon!
```

Comment:
This may be a question for later, but we may want this to always show when there's no open portfolio period, even if they have past portfolios. It may just work like this though, if the past portfolios are queried by user vs open portfolios getting queried by portfolio period.

Reply:
Honestly not sure if/when that happens. If it doesn't come up when there are closed periods, then it would also happen for shows. It's not too easy for me to check that since I already have active submission periods open for shows. I'll have to check on it once we have show deletion capabilities (or once I figure out how to do it through the graphql interface) I'll make a bugfix task to check on it.

Reply:
Unless you think we should just get this done now, it probably wouldn't be hard to figure out how to delete stuff through the graphql interface

Reply:
Ignore that I just figured out how to delete shows, its an issue for shows and portfolios.